specification/ai/Azure.AI.Projects/deployments/models.tsp

```diff
@@ -31,9 +31,9 @@ model Deployment {
   @visibility("read")
   modelVersion: string;
 
-  @doc("Name of the deployed model's publisher")
+  @doc("The publisher of the deployed model")
   @visibility("read")
-  modelPublisher: string;
+  publisher: string;
```

Comment:
same with this, I can change the API response to be `modelPublisher` instead of `publisher`. I'd rather it be `modelPublisher` since it makes more sense. Deployments themselves don't have publishers